build-system/server-inabox-friendly-template.html

```diff
@@ -0,0 +1,36 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>In-a-box Envelope</title>
+  <link rel="canonical" href="http://nonblocking.io/" >
+  <meta name="viewport" content="width=device-width,minimum-scale=1,initial-scale=1">
+  <style>
+    body {
+      margin: 0;
+    }
+    iframe {
+      border: 1px solid gray;
+    }
+  </style>
+</head>
+<body>
+  <h1>In-a-box Friendly Envelope</h1>
+  <div>url: AD_URL</div>
+  <div>size: AD_WIDTHxAD_HEIGHT</div>
+
+  <div style="background: #eee; overflow: hidden;; height: OFFSET;">scroll down to see the ad</div>
+
+  <iframe
```

Comment:
what about reusing server-inabox-template and make the whole `<iframe>` section dynamic?

Reply:
Changed it.